src/main/java/com/google/api/gax/core/Page.java

```diff
@@ -37,50 +39,26 @@
  * object also provides methods to retrieve additional pages using the page token, and to get the
  * API request and response objects.
  */
-public interface Page<RequestT, ResponseT, ResourceT> extends Iterable<ResourceT> {
+public interface Page<ResourceT> extends Iterable<ResourceT> {
```

Comment:
g-c-j has` iterateAll()` which avoids paging.  Are you saying that GAX would be that by default?  Or are we going back to a double loop again?  (ie. am I looking everything?  or am I having to do paging?)